app/RequestLoggingFilter.scala

```diff
@@ -0,0 +1,67 @@
+import akka.stream.Materializer
+import net.logstash.logback.marker.Markers.appendEntries
+import play.api.mvc.{Filter, RequestHeader, Result}
+import play.api.MarkerContext
+import utils.Logging
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.jdk.CollectionConverters._
+import scala.util.{Failure, Success}
+
+class RequestLoggingFilter(override val mat: Materializer)(implicit ec: ExecutionContext) extends Filter with Logging {
```

Comment:
I'm surprised it's required to implement this ourselves and there isn't some existing solution to produce access logs. Is it possible to get these from the ELB?

I'm presuming that you've thought of this and there is a good reason, so approving, but interested in the thought process.

Reply:
https://github.com/guardian/prism/pull/292 does the ELB based logging, however there are a couple of gaps there:
  - Where to define the bucket (aws-account-setup might be a good fit)
  - How to query the logs, as we don't have a route from S3 to Central ELK at the moment